modules/gdscript/gdscript.cpp

```diff
@@ -1246,40 +1246,43 @@ GDScript *GDScript::_get_gdscript_from_variant(const Variant &p_variant) {
 	return Object::cast_to<GDScript>(obj);
 }
 
+void GDScript::_get_function_dependencies(GDScriptFunction *p_func, RBSet<GDScript *> &p_dependencies, const GDScript *p_except) {
```

Comment:
Is `_get_function_dependencies` the right name? It actually inserts dependencies, isn't it?
(we know that you actually call the existing `scr->_get_dependencies()`, which inserts dependencies, hence the name)

Reply:
@rune-scape That's funny, I'm the one that named `_get_dependencies()`.

Reply:
Personally, I would expect `fetch` to be a returning (non-void) function too. For me `_collect_dependencies` or `_fill_dependencies` reflect better what happens.